modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx

```diff
@@ -54,7 +54,11 @@ import TaskHeaderActions from './TaskHeaderActions';
 
 type OutletContext = {
 	data: {
-		testrayTask: TestrayTask;
+		testrayTask: TestrayTask & {
+			actions: {
+				[key: string]: string;
+			};
+		};
```

Comment:
@rafaelfs98 add in the type `TestrayTask` this type: `actions: ObjectActionsItems`

See this example https://github.com/pt-liferay-solutions/liferay-portal/blob/3e876b037049874d545a8184acfb3b3461d6c59c/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/types.ts#L161